### ISSUE TITLE
Pull-up resistor 

### ISSUE THREAD

**Issue by SpectreDev-007** [folkhack/raspberry-pi-pwm-fan-2#1]
Is it possible for me to use 680ohm resistor instead, I am also using a noctua A4x10

**Comment by folkhack** [comment]
**TLDR:** I'm 99% sure you don't even need a pull-up resistor as-per my read of the Noctua PWM whitepaper but DriftKingTw's blog says a >660Ω pull-up resistor is required.

---

According to the Noctua whitepaper an external pull-up isn't required; but according to the blog I originally based the electronics on a pull-up resister > 660Ω is required to protect the fan.

* **Noctua Whitepaper:** "§ PWM control input signal [...] External pull-up is not necessary as the signal is pulled up to 3,3V/5V inside the fan." page 8 - https://noctua.at/pub/media/wysiwyg/Noctua_PWM_specifications_white_paper.pdf
* **DriftKingTw:** "The pull-up resistor that I use is 1kΩ 1/4W. The white paper says the maximum current is 5mA. According to Ohm’s Law [...], the resistor should be 660Ω or larger to protect your fan." - https://blog.driftking.tw/en/2019/11/Using-Raspberry-Pi-to-Control-a-PWM-Fan-and-Monitor-its-Speed/

I've been meaning to run a test without the pull-up as I'm pretty sure it's not necessary as-per the Noctua/Intel 2005 PWM spec. I'm deciding to keep this issue indefinitely open so people can weigh in with their experience/expertise =)

**Comment by SpectreDev-007** [comment]
Hmm ok thanks for that, another question I have is am I able to run 2 of them off 1 single 5v pin or will I have to wire them separately, (I am using the argon eon, I'm replacing the original fan since it's noisy which is a 60×60×10, so I need to replace it with 2 of the 40mm to compensate for the smaller size as the noctua 60mm doesn't fit)

**Comment by folkhack** [comment]
You should be fine running them off of the same 5V pin/PWM input signal but I have not tested that configuration.

**Comment by SpectreDev-007** [comment]
Thanks, I'll give it a shot and post an update on how it goes

**Comment by SpectreDev-007** [comment]
Just tried the 2 fan setup works flawlessly, I don't know if they're running at half the speed because it's 2 fans on one pin at full power but it works

**Comment by SpectreDev-007** [comment]
Also forgot to mention it seems to be working without disabling audio

**Comment by folkhack** [comment]
Closing issue as there is no applicable action at this time. Will investigate PWM working without disabling audio.
